cli-qpid-jms/src/test/kotlin/MainTest.kt

```diff
@@ -122,32 +125,25 @@ class AacMainTest : AbstractMainTest() {
 """.split(" ", "\n").toTypedArray()
 
     override fun main(args: Array<String>) = Main.main(args)
+
+    /**
+     * transport.logBytes option in qpid-jms
+     */
+    @Test
+    fun sendAndReceiveSingleMessageLogBytes() {
+        val senderParameters =
+            "sender --log-msgs dict --out json --broker $brokerUrl --address $address --count 1 --log-bytes".split(" ").toTypedArray()
+        val receiverParameters =
+            "receiver --log-msgs dict --out json --broker $brokerUrl --address $address --count 1 --log-bytes".split(" ").toTypedArray()
+        Assertions.assertTimeoutPreemptively(Duration.ofSeconds(10)) {
+            print("Sending: ")
+            main(senderParameters)
+            print("Receiving: ")
+            main(receiverParameters)
+        }
+    }
 }
 
-//class MainTest {
-//    @Test fun sendAndReceiveSingleMessage() {
-//        val senderParameters =
-//                "sender --log-msgs dict --broker amqp://127.0.0.1:5672 --address lalaLand --count 1".split(" ").toTypedArray()
-//        val receiverParameters =
-//                "receiver --log-msgs dict --broker amqpws://127.0.0.1:5672 --address lalaLand --count 1".split(" ").toTypedArray()
-////        print("Sending: ")
-////        Main.main(senderParameters)
-//        print("Receiving: ")
-//        Main.main(receiverParameters)
-//    }
-//
-//    @Test fun sendAndReceiveListMessage() {
```

Comment:
Which is reason to deletion test for list?

Reply:
It lives in https://github.com/rh-messaging/cli-java/blob/299a150b193f0b1d1aa4d2fe51dc9592b7931d05/tests/src/test/kotlin/AbstractMainTest.kt.

Reply:
not a problem